src/Jsons/Contributors.json

```diff
@@ -403,5 +403,14 @@
     "location": "Lagos, Nigeria 🇳🇬",
     "GitHub": "https://github.com/koleajeolayinka",
     "text": "I want to improve myself each day and help others do the same in a positive way."
-  }
+  },
+  {
+    "id": "Samejima",
+    "image": "img/bid_image/samejima.jpg",
+    "avatar": "img/avatar/samejima.jpg",
```

Comment:
Make sure your file path matches the images you upload in the `bid_image` and `avatar` file directories.